libffi/funcall.lisp

```diff
@@ -121,4 +120,6 @@
 
 (setf *foreign-structures-by-value* 'foreign-funcall-form/fsbv-with-libffi)
 
+(pushnew 'cffi-features:struct-by-value *features*)
```

Comment:
I suggest we just leave `:fsbv`, maybe with a deprecation note. (But perhaps @liamh feels differently.) We can definitely drop the `#+fsbv` from `cffi-tests` as I mentioned in another comment.